readme.md

```diff
@@ -366,6 +366,7 @@ Thanks for contributing! 🦋🙌
 - [](# "select-all-notifications-shortcut") Adds a shortcut to select all visible notifications: <kbd>a</kbd>.
 - [](# "linkify-notification-repository-header") [Linkifies the header of each notification group (when grouped by repository).](https://user-images.githubusercontent.com/1402241/80849887-81531c00-8c19-11ea-8777-7294ce318630.png)
 - [](# "stop-redirecting-in-notification-bar") [Stops redirecting to notification inbox from notification bar actions while holding <kbd>Alt</kbd>.](https://user-images.githubusercontent.com/202916/80318782-c38cef80-880c-11ea-9226-72c585f42a51.png)
+- [](# "linkify-notification-page-numbers") [Linkifies the page number of notifications to jump to the last page.](https://user-images.githubusercontent.com/16872793/199788308-0f4d8fa2-9a80-4049-9f43-60a9f79c3b80.png)
```

Comment:
The feature is "linking to the last page;" linkification is just how we achieve that. 

`last-notification-page-button` is more helpful, but long